packages/workbox-precaching/utils/printCleanupDetails.mjs

```diff
@@ -7,47 +7,32 @@
 */
 
 import {logger} from 'workbox-core/_private/logger.mjs';
+
 import '../_version.mjs';
 
-const logGroup = (groupTitle, urls) => {
+const logGroup = (groupTitle, deletedUrls) => {
   logger.groupCollapsed(groupTitle);
 
-  urls.forEach((url) => {
+  for (const url of deletedUrls) {
     logger.log(url);
-  });
+  }
 
   logger.groupEnd();
 };
 
 /**
- * @param {Array<string>} deletedCacheRequests
- * @param {Array<string>} deletedRevisionDetails
+ * @param {Array<string>} deletedUrls
  *
  * @private
- * @memberof module:workbox-precachig
+ * @memberof module:workbox-precaching
  */
-export default (deletedCacheRequests, deletedRevisionDetails) => {
-  if (deletedCacheRequests.length === 0 &&
-    deletedRevisionDetails.length === 0) {
-    return;
+export default (deletedUrls) => {
+  const deletionCount = deletedUrls.length;
+  if (deletionCount > 0) {
```

Comment:
Should this all be inside a `NODE_ENV !== 'production'` conditional?

Reply:
The code that calls this module is inside a `NODE_ENV` check.